common/src/types/v0/store/replica.rs

```diff
@@ -93,6 +93,92 @@ pub struct ReplicaSpec {
     pub operation: Option<ReplicaOperationState>,
 }
 
+/// Reference of a pool.
+#[derive(Serialize, Deserialize, Debug, Clone, Eq, PartialEq)]
+#[serde(untagged)]
+pub enum PoolRef {
+    /// Name referenced pool.
+    Named(PoolId),
+    /// Uuid and name referenced pool.
+    Uuid(PoolId, PoolUuid),
+}
+
+impl Default for PoolRef {
+    fn default() -> Self {
+        Self::Named("".into())
+    }
+}
```

Comment:
Instead you might be able to add #[default] on top of named in the enum definition.

Reply:
For enum, it errors out. We have you define our own for enum definition.

Reply:
Ah, looks like it only works for unit enum variants.

Reply:
yes.